packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -196,7 +196,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     // If the returned value is not an array then check to see if it's a promise or PromiseLike. If it is then resolve it asynchronously.
     if (Array.isArray(suggestionsArray)) {
       this.resolveNewValue(updatedValue, suggestionsArray);
-    } else if (suggestionsPromiseLike.then) {
+    } else if (suggestionsPromiseLike) {
```

Comment:
@Scott-Cochrane the TitleCased directories are to match ClassNames that are contained within; other folders (like app, packages, etc) are camelCased. Because "pickers" isn't a class... that's the reasoning.